My task ID: 33
Number of Tasks: 96

TRAINING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 100 | 
GPR | 100 | 
GRP | 100 | 
PGR | 100 | 
PPP | 100 | 
PRG | 100 | 
RGP | 100 | 
RPG | 100 | 
RRR | 100 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 50 | 
GGR | 50 | 
GPG | 50 | 
GPP | 50 | 
GRG | 50 | 
GRR | 50 | 
PGG | 50 | 
PGP | 50 | 
PPG | 50 | 
PPR | 50 | 
PRP | 50 | 
PRR | 50 | 
RGG | 50 | 
RGR | 50 | 
RPP | 50 | 
RPR | 50 | 
RRG | 50 | 
RRP | 50 | 

----------

TESTING DATA

Accepting Grid:
SET_combinations | Number of Trials | Status
GGG | 5 | 
GPR | 5 | 
GRP | 5 | 
PGR | 5 | 
PPP | 5 | 
PRG | 5 | 
RGP | 5 | 
RPG | 5 | 
RRR | 5 | 

Rejecting Grid:
SET_combinations | Number of Trials | Status
GGP | 5 | 
GGR | 5 | 
GPG | 5 | 
GPP | 5 | 
GRG | 5 | 
GRR | 5 | 
PGG | 5 | 
PGP | 5 | 
PPG | 5 | 
PPR | 5 | 
PRP | 5 | 
PRR | 5 | 
RGG | 5 | 
RGR | 5 | 
RPP | 5 | 
RPR | 5 | 
RRG | 5 | 
RRP | 5 | 

----------

GROK DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status

----------

CORRUPT DATA

Accepting Grid:
SET_combinations | Number of Trials | Status

Rejecting Grid:
SET_combinations | Number of Trials | Status


Metrics after epoch 50:
train_loss: 1.0015679597854614
train_accuracy: 0.0
test_loss: 1.0017071962356567
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 100:
train_loss: 0.9965757131576538
train_accuracy: 0.0
test_loss: 1.0091644525527954
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 150:
train_loss: 0.9796057939529419
train_accuracy: 0.0
test_loss: 0.9910351037979126
test_accuracy: 0.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 200:
train_loss: 0.10542242974042892
train_accuracy: 0.8152901530265808
test_loss: 0.1145092323422432
test_accuracy: 0.940740704536438
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 250:
train_loss: 0.016752520576119423
train_accuracy: 0.9737723469734192
test_loss: 0.017963912338018417
test_accuracy: 0.9925925731658936
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 300:
train_loss: 0.002181256888434291
train_accuracy: 1.0
test_loss: 0.002953136805444956
test_accuracy: 0.9925925731658936
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 350:
train_loss: 0.0030926051549613476
train_accuracy: 0.9972098469734192
test_loss: 0.01360353734344244
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 400:
train_loss: 0.0008627938805148005
train_accuracy: 1.0
test_loss: 0.001036035013385117
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 450:
train_loss: 0.00043401887523941696
train_accuracy: 1.0
test_loss: 0.00046830190694890916
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 500:
train_loss: 0.0024278375785797834
train_accuracy: 0.99609375
test_loss: 0.0013913001166656613
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 550:
train_loss: 0.00040172148146666586
train_accuracy: 1.0
test_loss: 0.0003677355998661369
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 600:
train_loss: 0.0007792561664246023
train_accuracy: 1.0
test_loss: 0.00036833598278462887
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 650:
train_loss: 0.00035860532079823315
train_accuracy: 1.0
test_loss: 0.013810344971716404
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 700:
train_loss: 0.0001271705114049837
train_accuracy: 1.0
test_loss: 0.00012934586266055703
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 750:
train_loss: 0.0003293009358458221
train_accuracy: 1.0
test_loss: 0.0051412503235042095
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 800:
train_loss: 0.0022315236274152994
train_accuracy: 0.9955357313156128
test_loss: 0.01317841000854969
test_accuracy: 0.9629629254341125
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 850:
train_loss: 0.00016181137470994145
train_accuracy: 1.0
test_loss: 0.0004044789238832891
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 900:
train_loss: 5.78306608076673e-05
train_accuracy: 1.0
test_loss: 6.0147962358314544e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 950:
train_loss: 3.8644247979391366e-05
train_accuracy: 1.0
test_loss: 4.1271527152275667e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


Metrics after epoch 1000:
train_loss: 2.68486728600692e-05
train_accuracy: 1.0
test_loss: 3.5029683203902096e-05
test_accuracy: 1.0
grok_loss: None
grok_accuracy: None
corrupt_loss: None
corrupt_accuracy: None


LLSUB_RANK=33: 2023-09-20 11:49:15.765136: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
LLSUB_RANK=33: 2023-09-20 11:49:19.475162: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=33: 2023-09-20 11:49:19.475284: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
LLSUB_RANK=33: 2023-09-20 11:49:19.475291: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
LLSUB_RANK=33: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:98: UserWarning: unable to load libtensorflow_io_plugins.so: unable to open file: libtensorflow_io_plugins.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so']
LLSUB_RANK=33: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so: undefined symbol: _ZN3tsl6StatusC1EN10tensorflow5error4CodeESt17basic_string_viewIcSt11char_traitsIcEENS_18SourceLocationImplE']
LLSUB_RANK=33:   warnings.warn(f"unable to load libtensorflow_io_plugins.so: {e}")
LLSUB_RANK=33: /state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/__init__.py:104: UserWarning: file system plugins are not loaded: unable to open file: libtensorflow_io.so, from paths: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so']
LLSUB_RANK=33: caused by: ['/state/partition1/llgrid/pkg/anaconda/anaconda3-2023a-tensorflow/lib/python3.9/site-packages/tensorflow_io/python/ops/libtensorflow_io.so: undefined symbol: _ZTVN3tsl13GcsFileSystemE']
LLSUB_RANK=33:   warnings.warn(f"file system plugins are not loaded: {e}")
LLSUB_RANK=33: No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
LLSUB_RANK=33: 2023-09-20 11:50:26.086302: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
LLSUB_RANK=33: 2023-09-20 11:50:26.086331: W tensorflow/compiler/xla/stream_executor/cuda/cuda_driver.cc:265] failed call to cuInit: UNKNOWN ERROR (303)
